lesson_03/script.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+const ul = document.getElementById("js-target");
+const data = ["a1.html", "a2.html"];
+const fragment = document.createDocumentFragment();
+for (let i = 1; i <= data.length; i += 1) {
+  const li = document.createElement("li");
+  const a = document.createElement("a");
+  const img = document.createElement("img");
+  a.text = `a${i}`;
+  a.href = `${data[i - 1]}`;
```

Comment:
```suggestion
  a.href = data[i-1];
```
IMO
There is no need to enclose it in a template literal, so it is easier to see if you remove it.